cmd/mesh/root.go

```diff
@@ -28,6 +28,8 @@ const (
 Overrides the corresponding path value in the selected profile or passed through IstioControlPlane CR
 customization file`
 	filenameFlagHelpStr = `Path to file containing IstioControlPlane CustomResource`
+	yesFlagHelpStr      = `Do not ask for confirmation when manifest apply. If --yes or -y flag is not set, its default value is false. 
+If --yes or -y flag is set without any options, then its default value is true`
 )
```

Comment:
@ayj Here I explained how --yes flag being used and what's the default value with or without --yes flag. I agree, this flag is really confusing.  Let me think how to make it better for usage.